networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc

```diff
@@ -61,7 +61,7 @@ The Ingress Operator manages wildcard DNS. For more information, see the followi
 
 * xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a cluster on bare metal].
 
-* xref:../../installing/installing_vsphere/installing-vsphere.adoc#installing-vsphere[Installing a cluster on vSphere].
+* xref:../../installing/installing_vsphere/upi/installing-vsphere.adoc#installing-vsphere[Installing a cluster on vSphere].
```

Comment:
I doubled checked this and it works, but I'm not sure why you wouldn't have to go three levels deep.